scripts/load_testing/create_account.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+from faker import Factory
+import locust
+import pyquery
+
+fake = Factory.create()
+
+username, password = os.getenv('AUTH_USER'), os.getenv('AUTH_PASS')
+auth = (username, password) if username and password else ()
+
+class UserBehavior(locust.TaskSet):
+    @locust.task
+    def signup(self):
+        # visit home page
+        self.client.get('/', auth=auth)
+
+        # visit create account page and submit email
+        resp = self.client.get('/sign_up/enter_email', auth=auth)
+        resp.raise_for_status()
+        dom = pyquery.PyQuery(resp.content)
+        auth_token = dom.find('input[name="authenticity_token"]')[0].attrib['value']
+        data = {
+            'user[email]': 'test+' + fake.md5() + '@test.com',
+            'authenticity_token': auth_token,
+            'commit': 'Submit',
+        }
+        resp = self.client.post('/sign_up/register', data=data, auth=auth)
+        resp.raise_for_status()
+
+        # capture email confirmation link on resulting page
+        dom = pyquery.PyQuery(resp.content)
+        link = dom.find('a')[2].attrib['href']
+
+        # click email confirmation link and submit password
+        resp = self.client.get(link, auth=auth)
+        resp.raise_for_status()
+        dom = pyquery.PyQuery(resp.content)
+        auth_token = dom.find('input[name="authenticity_token"]')[0].attrib['value']
+        confirmation_token = dom.find('input[name="confirmation_token"]')[0].attrib['value']
+        data = {
+            'password_form[password]': 'salty pickles',
+            'authenticity_token': auth_token,
+            'confirmation_token': confirmation_token,
+            'commit': 'Submit',
+        }
+        resp = self.client.post('/sign_up/create_password', data=data, auth=auth)
+        resp.raise_for_status()
+
+        # visit phone setup page and submit phone number
+        dom = pyquery.PyQuery(resp.content)
+        auth_token = dom.find('input[name="authenticity_token"]')[0].attrib['value']
+        data = {
+            'two_factor_setup_form[phone]': '7035550001',
+            'two_factor_setup_form[otp_method]': 'sms',
+            'authenticity_token': auth_token,
+            'commit': 'Send passcode',
+        }
+        resp = self.client.patch('/phone_setup', data=data, auth=auth)
+        resp.raise_for_status()
+
+        # visit enter passcode page and submit pre-filled OTP
+        dom = pyquery.PyQuery(resp.content)
+        auth_token = dom.find('input[name="authenticity_token"]')[0].attrib['value']
```

Comment:
Looks like we repeat the authenticity token blurb a few times, WDYT of a helper function for authenticity token?

```python
def authenticity_token(dom):
  return dom.find('input[name="authenticity_token"]')[0].attrib['value']

data = {
  'code': otp_code,
  'authenticity_token': authenticity_token(dom),
  'commit': 'Submit'
}
```